test/unit/APIConsumer_unit_test.js

```diff
@@ -4,14 +4,15 @@ const chai = require('chai')
 const { expect } = require('chai')
 const BN = require('bn.js')
 const { getChainId } = require('hardhat')
+const {numToBytes32} = require("@chainlink/test-helpers/dist/src/helpers");
 chai.use(require('chai-bn')(BN))
 
 skip.if(!developmentChains.includes(network.name)).
   describe('APIConsumer Unit Tests', async function () {
 
-    let apiConsumer, linkToken
+    let apiConsumer, linkToken, mockOracle
 
-    beforeEach(async () => {
+    before(async () => {
```

Comment:
We tend to make each test a separate unit so I would suggest to leave `beforeEach` hook here

```suggestion
    beforeEach(async () => {
```